test/runners/python_spec.js

```diff
@@ -133,6 +133,28 @@ describe('python runner', function() {
           done();
         });
       });
+      it('should show KeyErrors', function(done) {
+        runner.run({
+          language: 'python',
+          languageVersion: lv,
+          testFramework: 'unittest',
+          code: 'a = {}; a[10] += 3',
```

Comment:
@Dan-Nolan these `testFramework` and `code` needs to be removed because ESLint is failing with "Duplicate key".

Reply:
Ah thanks! I'll fix this up.